Console.rb

```diff
@@ -0,0 +1,61 @@
+# Utility for writing to terminal, mostly colors and cursor movements.
+
+module Console
+	def self.color(color, text)
+		return text if color.nil?
+		self.public_send(color, text)
+	end
```

Comment:
I've been burned by this so many times that I just enable whitespace glyphs all the time http://sublimetexttips.com/show-whitespace-sublime-text/